builder/qemu/builder.go

```diff
@@ -534,7 +544,11 @@ func (b *Builder) Prepare(raws ...interface{}) ([]string, []string, error) {
 			errs, fmt.Errorf("vnc_port_min must be less than vnc_port_max"))
 	}
 
-	if b.config.VNCUsePassword && b.config.QMPSocketPath == "" {
+	if b.config.NetBridge != "" || b.config.VNCUsePassword {
```

Comment:
In the Prepare() we should probably do a validation check on the operating system to make sure it's a linux-based OS and fail early if someone is trying to run on windows or macOS

Reply:
Ah indeed, I will validate it!

This could probably work in other OSes, but currently, I'm only able to test in Linux/Ubuntu.

Reply:
Its now validating.